client/reducers/error-message.js

```diff
@@ -0,0 +1,13 @@
+import {SHOW_ERROR} from '../actions'
```

Comment:
hmm, not sold but could be convinced. I get that they might save students from misnaming action types but I haven't seen many errors caused by that and this is one more thing for people to parse when getting their head around redux.